src/components/button-toggle/button-toggle.style.ts

```diff
@@ -126,6 +128,31 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
       }
       cursor: not-allowed;
     `};
+
+  ${({ variant }) =>
+    variant === "minor" &&
```

Comment:
**issue**: a check against the `disabled` prop should be added here. To stop any styling being added when `disabled` is true. If you hover on a `disabled` button with the minor variant, the hover colors are still applied

Reply:
There are some differences between the two variants fro example the border-radius is smaller when `minor` is set, it would probably work better to move the block setting the `disabled` styling is applied after the `minor` styling